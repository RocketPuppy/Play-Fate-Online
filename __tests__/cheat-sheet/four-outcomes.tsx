import * as React from "react";
import * as renderer from "react-test-renderer";
import FourOutcomes from "../../src/cheat-sheet/four-outcomes";

test("renders all the pieces", () => {
  const render = renderer.create(<FourOutcomes />);

  expect(render.toJSON()).toMatchSnapshot();
});
