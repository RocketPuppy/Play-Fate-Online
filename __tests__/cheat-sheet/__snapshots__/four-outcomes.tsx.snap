// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Four Outcomes
  </h2>
  <dl>
    <dt>
      Fail
    </dt>
    <dd>
      fail your action or succeed at major cost
    </dd>
    <dt>
      Tie (0)
    </dt>
    <dd>
      succeed at minor cost
    </dd>
    <dt>
      Succeed (1â€”2)
    </dt>
    <dd>
      succeed with no cost
    </dd>
    <dt>
      Succeed with Style (&gt;3)
    </dt>
    <dd>
      succeed with additional benefit
    </dd>
  </dl>
</div>
`;
